Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -4,30 +4,30 @@ import MapboxDirections
 /// Defines source of routing engine (online or offline) to be used for requests.
 public typealias RoutingProviderSource = MapboxRoutingProvider.Source
 
-/// Configures Navigator status polling options.
-public struct StatusPollingConfig {
+/// Configures Navigator status polling.
+public struct StatusUpdatingSettings {
     /**
-     If new location is not provided during `unconditionalPollingPatience` - status will be polled unconditionally.
+     If new location is not provided during `updatePatience` - status will be polled unconditionally.
      
      If `nil` - default value will be used.
      */
-    public var unconditionalPollingPatience: TimeInterval?
+    public var updatePatience: TimeInterval?
```

Comment:
This should be `updatingPatience`, since otherwise it could sound like a method name. Likewise with `updatingInterval`.

Reply:
Thanks, it has been updated